misc/util.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+##
+# Miscellaneous utility functions
+
+##
+# Set the timezone
+# param1 - Timezone to set. Defaults to "UTC"
+#          Eg: "US/Mountain", "America/Los_Angeles" etc.
+#
+# This function sets the timezone on the cluster node.
+# After setting the timezone, it is advised to restart engine daemons on the master and worker nodes
+#
+function set_timezone() {
+    timezone=${1:-UTC}
```

Comment:
IMO changing to UTC doesn't make sense since we run in UTC by default. It would be better to make the timezone a mandatory parameter and error out if it's not provided.

Reply:
Done. Also added check to see that a valid timezone was provided